docs/adr/20230119-use-the-term-decisions-instead-of-adr.md

```diff
@@ -0,0 +1,27 @@
+# Use the term decisions instead of adr
+
+- Status: accepted
+- Deciders: Tyler Hubert, Thomas Vaillant, Joel Parker Henderson
+
+Technical Story: https://github.com/thomvaill/log4brains/issues/12
+
+## Context and Problem Statement
+
+The idea behind documenting any decision that gets made is becoming more popular.  We should update the terminology in this library to reflect that any decision, not just architectural ones, should be documented and that this library can make that easier.
+
+## Decision Drivers
+
+- ADRs are starting to update their terminology to be Any Decision Records instead of Architectural, so using the more generic term decision is a simpler term that more people will understand.
```

Comment:
Note that we intend to switch back to "architectural" to strengthen the importance of "architecture". See our CHANGELOG of the upcoming release at https://github.com/adr/madr/blob/develop/CHANGELOG.md

Any feedback welcome!